impl/src/settings.rs

```diff
@@ -531,6 +531,45 @@ impl RawRazeSettings {
       || self.rust_rules_workspace_name.is_some()
       || self.vendor_dir.is_some()
   }
+
+  fn print_deprecation_warnings(&self) {
+    if self.incompatible_relative_workspace_path.is_some() {
+      eprintln!(
+        "WARNING: `[*.raze.incompatible_relative_workspace_path]` will soon be deprecated. Please \
+         update your project to use the `true` behavior."
+      );
+    }
+
+    if self.target.is_some() {
+      eprintln!(
+        "WARNING: `[*.raze.target]` will soon be deprecated. Please update your project to use \
+         `[*.raze.targets]`."
+      );
+    }
+
+    if self.default_gen_buildrs.is_none() {
+      eprintln!(
+        "WARNING: The default of `[*.raze.default_gen_buildrs]` will soon be set to `true`. \
```

Comment:
Hmm. I don't know about this one. What you're doing here is basically making the flag mandatory.

Perhaps this should be a "notice", instead of a "warning".

Reply:
I can change this to `NOTICE:`. The inspiration for adding this is in https://github.com/bazelbuild/rules_rust/issues/490

Reply:
And It's not making it mandatory, I'm just trying to inform users that behavior will change

Reply:
Updated